README.md

```diff
@@ -212,15 +205,20 @@ for i, pose in poses_df.iterrows():
 
 ### Camera rig ROS JSON file
 
-Camera rig JSON files contain spacial data that uses the same coordinate system described in the `Camera pose output` section. It contains 5 fields for each rig camera:
+Camera rig JSON files contain spatial data that includes 4 fields for each rig camera:
 
 - `intrinsics` - Camera intrinsics matrix
-- `coord_sys` - Type of the coordinate system, `RDF` is required
 - `rotation` - Rotation quaternion relative to the rig origin
 - `translation` - Translation vector relative to the rig origin
 - `sensor_size` - Sensor width and height in pixels, used to calculate camera FOV
 
-Following is a ROS JSON file example for a rig with two parallel cameras with the FOV of 90 degrees, facing forward. The difference can be found in the sign of the translation vector X-axis.
+The used coordinate system axis orientations are:
+
+- X-axis points forward (along the camera view direction)
```

Comment:
This is kind of misleading. Extrinsics (rotation and translation) are in rig reference frame which is initially in the origin with no rotation. I think it is sufficient to say that the axes correspond to the UE world axes.

Reply:
Will clarify after the merge